src/qtwebengine/dataprovider.cpp

```diff
@@ -41,8 +97,8 @@ void DataProvider_QWebEngine::requestStarted( QWebEngineUrlRequestJob *request )
     outbuf->close();
 
     // Only delete the buffer when the request is deleted too
-    connect( request, SIGNAL(destroyed()), outbuf, SLOT(deleteLater()) );
+    connect( request, SIGNAL( destroyed() ), outbuf, SLOT( deleteLater() ) );
```

Comment:
Please no. Any refactoring/reformatting MUST be a separate pull request, and should be discussed in advance if you want to have them integrated.

Reply:
Sorry, it was an accident. Apparently, it happened until I disabled autoformatting in the IDE.